Stock News Sentiment Analysis/Model/README.md

```diff
@@ -0,0 +1,2 @@
+# Stock News Sentiment Analysis
```

Comment:
Please follow the `README.md` template and update it accordingly. Here is the template, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md